ecl/eclcmd/eclcmd_core.cpp

```diff
@@ -886,13 +886,31 @@ class EclCmdRun : public EclCmdWithEclTarget
 
         switch (state)
         {
-        case WUStateRunning:
-            fprintf(stderr, "Timed out waiting for %s to complete, workunit is still running.\n", wuid.str()); //server side waiting timed out
-            break;
         case WUStateCompleted:
             break;
+        case WUStateCompiled:
```

Comment:
This command is only for running workunits, so is never used to request a compile.
Just a compile can be done via the "deploy" command which has different result handling logic than this API call.